models/agol.js

```diff
@@ -990,6 +990,7 @@ var AGOL = function (koop) {
           if (err) agol.log.error(err)
           try {
             error = JSON.parse(value)
+            if (error.type === 'db') return removeJob(job)
```

Comment:
That's actually the point here.

This error occurs when partial inserts happen after a table has been dropped from the cache after a /drop call. The best thing to do here is just ignore the failed insert. Otherwise you get stuck with a failed dataset instead of no dataset which is what you wanted.